src/users/users.service.ts

```
@@ -232,7 +232,7 @@ export class UsersService {
 
       return {};
     } catch (error) {
-      throw new NotFoundException(MSG.EMAIL_NOT_FOUND);
+      throw new BadRequestException();
```

You are swallowing original error. At least log it. Swallowing errors leads to long unhappy hours of debugging in future